third_party/conan/lockfiles/linux/clang7_debug/conan.lock

```diff
@@ -574,174 +570,174 @@
     "options": ""
    },
-   "72": {
+   "70": {
     "pref": "cmake_installer/3.16.3@conan/stable#0:44fcf6b9a7fb86b2586303e3db40189d3b511830",
     "options": ""
    },
-   "73": {
+   "71": {
     "pref": "cmake_installer/3.16.3@conan/stable#0:44fcf6b9a7fb86b2586303e3db40189d3b511830",
     "options": ""
    },
-   "74": {
+   "72": {
     "pref": "cmake_installer/3.16.3@conan/stable#0:44fcf6b9a7fb86b2586303e3db40189d3b511830",
     "options": ""
    },
-   "75": {
+   "73": {
     "pref": "cmake_installer/3.16.3@conan/stable#0:44fcf6b9a7fb86b2586303e3db40189d3b511830",
     "options": ""
    },
-   "76": {
+   "74": {
     "pref": "protoc_installer/3.9.1@bincrafters/stable#0:c0c1ef10e3d0ded44179e28b669d6aed0277ca6a",
     "options": "protobuf:fPIC=True\nprotobuf:lite=False\nprotobuf:shared=False\nprotobuf:with_zlib=False",
     "requires": [
-     "77"
+     "75"
     ],
     "build_requires": [
-     "80"
+     "78"
     ]
    },
-   "77": {
+   "75": {
     "pref": "protobuf/3.9.1@bincrafters/stable#0:8af60a965d2b176751abd5c14af8f4f7530f2c71",
     "options": "fPIC=True\nlite=False\nshared=False\nwith_zlib=False",
     "build_requires": [
-     "79"
+     "77"
     ]
    },
-   "78": {
+   "76": {
     "pref": "grpc_codegen/1.27.3@orbitdeps/stable#ec39b3cf6031361be942257523c1839a:0b6eee9c211529a5558295428b69441f41a01ae6",
     "options": "fPIC=True\nprotobuf:fPIC=True\nprotobuf:lite=False\nprotobuf:shared=False\nprotobuf:with_zlib=False",
     "requires": [
-     "76"
+     "74"
     ],
     "build_requires": [
+     "79",
      "81",
+     "82",
      "83",
      "84",
-     "85",
-     "86",
-     "93"
+     "91"
     ]
    },
-   "79": {
+   "77": {
     "pref": "cmake_installer/3.16.3@conan/stable#0:44fcf6b9a7fb86b2586303e3db40189d3b511830",
     "options": ""
    },
-   "80": {
+   "78": {
     "pref": "cmake_installer/3.16.3@conan/stable#0:44fcf6b9a7fb86b2586303e3db40189d3b511830",
     "options": ""
    },
-   "81": {
+   "79": {
     "pref": "abseil/20190808@orbitdeps/stable#0:4b2b9332d1eebd37c99ceaab3ea70e9fb0527815",
     "options": "cxx_standard=17\nfPIC=True\ncctz:build_tools=False\ncctz:fPIC=True\ncctz:shared=False",
     "requires": [
-     "82"
+     "80"
     ],
     "build_requires": [
-     "91"
+     "89"
     ]
    },
-   "82": {
+   "80": {
     "pref": "cctz/2.3#0:8af60a965d2b176751abd5c14af8f4f7530f2c71",
     "options": "build_tools=False\nfPIC=True\nshared=False",
     "build_requires": [
-     "88"
+     "86"
     ]
    },
-   "83": {
+   "81": {
     "pref": "zlib/1.2.11#9e0c292b60ce77402bd9be60dd68266f:717ad2d3c8b08dce4dcc7e8e6a9291bfc044d7e9",
     "options": "fPIC=True\nminizip=False\nshared=False",
     "build_requires": [
-     "90"
+     "88"
     ]
    },
-   "84": {
-    "pref": "openssl/1.0.2t#a108e98ed2578863c45746ff2bd2512d:fc1522b9e4333a8c74ca8b19c266b1939a89f941",
+   "82": {
+    "pref": "openssl/1.0.2t#146f912909a48f790ee66ca0cc3716b6:fc1522b9e4333a8c74ca8b19c266b1939a89f941",
```

Comment:
This openssl change shouldn't be here. Can you revert the change of the recipe revision? Doing that manually is probably the simplest solution.

Reply:
Unfortunately this has to be done twice per file for all lockfiles.

Reply:
Fixed, thanks for noticing!